tools/rust_api/src/kuzu_rs.cpp

```diff
@@ -19,27 +19,26 @@ std::unique_ptr<QueryParams> new_params() {
     return std::make_unique<QueryParams>();
 }
 
-std::unique_ptr<LogicalType> create_logical_type(kuzu::common::LogicalTypeID id) {
-    return std::make_unique<LogicalType>(id);
+LogicalType create_logical_type(kuzu::common::LogicalTypeID id) {
```

Comment:
@benjaminwinger should review rust part

Reply:
I'm keeping unique_ptr in rust for now since I don't know how to remove it. I think Ben should do this part.